web/web.go

```diff
@@ -132,19 +131,17 @@ func New(opts Options) (*Engine, error) {
 	return e, nil
 }
 
-// StartAsync HTTP with custom address and return stop channel.
-func StartAsync(e *Engine, bind string) <-chan os.Signal {
-	ch := make(chan os.Signal, 1)
-	signal.Notify(ch, syscall.SIGINT, syscall.SIGQUIT, syscall.SIGTERM, syscall.SIGABRT)
-
+// StartAsync HTTP with custom address.
+func StartAsync(e *Engine, bind string, cancel context.CancelFunc) {
```

Comment:
O_o.. we've discussed this.. it was a completely different story..

Reply:
`StartAsync(*Engine, bind string) return func(Context) error`...

Reply:
than this option is bad?